feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchActivity.kt

```diff
@@ -80,16 +92,23 @@ class SearchActivity : ComponentActivity() {
     }
   }
 
-  private fun startClaimsFlow(viewModel: SearchViewModel, commonClaimId: String?) {
-    val intent = OdysseyClaimsFlowActivity.newInstance(
-      context = this,
-      odysseyUrl = odysseyUrl,
-      commonClaimId = commonClaimId,
-    )
+  private suspend fun startClaimsFlow(viewModel: SearchViewModel, commonClaimId: String?) {
```

Comment:
Maybe not in the scope of this PR, maybe in the future we can consider removing passing the entire viewModel here, but instead pass a lambda to call the viewmodel would be a bit nicer.